### PR TITLE
wireguard: use the new DNS setting

### PR DESCRIPTION
**Pull request by zx2c4**
This came out with 0.0.20170726, and merging it probably should depend on @EggieCode pushing the package to the Ubuntu PPA.

### CODE REVIEW

File: playbooks/roles/wireguard/templates/instructions-fr.md.j2 [hunk 1]
```diff
@@ -28,4 +28,4 @@ WireGuard n'est actuellement disponible que pour Linux, mais support [multi-plat
    `sudo wg-quick down wg0-client`
 
 #### Une note sur la configuration DNS ####
-Le fichier `wg0-client.conf` inlcure des commandes `PostUp` et `PostDown` simples qui utilisent `resolvconf` afin de diriger le trafic DNS vers le serveur dnsmasq qui est disponible via l'interface cryptée WireGuard à `{{ dnsmasq_wireguard_ip }}`. Bien que `resolvconf` soit un utilitaire commun, vous devrez peut-être modifier ces lignes pour votre distribution ou votre configuration réseau particulière.
+Le fichier `wg0-client.conf` inlcure le commande `DNS` qui utilise `resolvconf` afin de diriger le trafic DNS vers le serveur dnsmasq qui est disponible via l'interface cryptée WireGuard à `{{ dnsmasq_wireguard_ip }}`. Bien que `resolvconf` soit un utilitaire commun, vous devrez peut-être remplacer ce ligne avec `PostUp`/`PostDown` pour votre distribution ou votre configuration réseau particulière.
```
Comment (alimakki) [inline playbooks/roles/wireguard/templates/instructions-fr.md.j2]: le commande -> la commande
...
remplacer ce ligne avec -> remplacer cette ligne avec

Comment (zx2c4) [inline playbooks/roles/wireguard/templates/instructions-fr.md.j2]: Indeed I have no idea the gender of any nouns. Sometimes when I speak I just kind of blur the articles into a halfway sound and hope nobody notices my laziness.

Will fix and re-push.

**Comment by EggieCode** [comment]
Packages are uploaded to the PPA!
https://launchpad.net/~wireguard/+archive/ubuntu/wireguard/+packages

**Comment by alimakki** [comment]
Thanks you all

**Pull request merged**